test/README.md

```diff
@@ -126,3 +130,243 @@ and `setInterval`).
 | Runs on CI |
 |:----------:|
 | No         |
+
+
+## Common.js
+
+### allowGlobals(...whitelist)
+* `whitelist` [&lt;Array>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) Array of Globals
```

Comment:
For this I used similar formatting to https://nodejs.org/dist/latest-v6.x/docs/api/fs.html#fs_event_change